debug_toolbar/settings.py

```diff
@@ -37,6 +37,7 @@
         "django.utils.deprecation",
         "django.utils.functional",
     ),
+    "HISTORY_POST_TRUNC_LENGTH": 0,
```

Comment:
Nitpick: I think `TRUNCATE` should be spelled out and not abbreviated. Except for `MAX` no other key contains an abbreviated word currently.

Reply:
Makes sense.

Reply:
I removed this setting entirely. Instead we're going to show a table. There's a risk in the results cache growing too large, but I'd rather not include settings if possible.